docs/src/whatsnew/1.7.rst

```diff
@@ -329,6 +329,6 @@ Documentation
 * A clarification of the behaviour of
   :func:`iris.analysis.calculus.differentiate`.
 
-* A new :doc:`"Technical Papers" </techpapers/index>` section has been added to
+* A new Technical Papers section has been added to
```

Comment:
This is the 1.7 what's new; whereas all I had done was move where the Tech Papers section was, now I've removed it altogether. To change the path to further topics seems non-representative. 

The file I've linked to because the file hasn't changed.